components/FileUploader.js

```diff
@@ -35,6 +39,11 @@ class FileUploaderButton extends Component {
     }
   }
 
+  handleClick = evt => {
+    console.log('click');
```

Comment:
It's ready to review now, but looks like I still have to go back and cleanup some things like this! Thank you